lib/jobs/run-work-items.js

```diff
@@ -207,6 +208,37 @@ function runWorkItemsJobFactory(
             );
         });
     };
+    
+    /**
+     * @function _executeRedfishPoller
+     * @description execute the redfish worker item 
+     */
+    RunWorkItemsJob.prototype._executeRedfishPoller = function (workItem) {
+        assert.object(workItem.config, 'workItem.config');
+        var self = this;
+
+        return Promise.resolve()
+        .then(function() {
+            assert.ok(workItem.config,'workItem.config');
+            assert.string(workItem.config.command, 'workItem.config.command');
```

Comment:
Duplicated assertion with line#217 and why not move these assertion together with line#217?